App/Components/Chat/CompChatView.razor

```diff
@@ -168,73 +269,152 @@
 @* ReSharper disable always InconsistentNaming *@
 @code {
     [Inject] IUserGroupService UserGroupService { get; set; } = default!;
-    
+
     [CascadingParameter] public EventCallback<KnownPopupArgs> OpenKnownPopup { get; set; }
-    
-    [Parameter] public int? ChatID { get; set; } 
-    public List<int?> WatchedIDs { get; set; } = [];
+    [CascadingParameter] public EventCallback ClosePopupCallback { get; set; }
+    [CascadingParameter] public EventCallback<IPacket> SendToServer { get; set; }
+    [CascadingParameter] public Predicate<IAsyncPacketSubscriber> AddPacketSubscriber { get; set; }
+    [CascadingParameter] public Predicate<IAsyncKeySubscriber> AddKeySubscriber { get; set; }
+
+    [Parameter] public int? ChatID { get; set; }
     private Chat? Chat { get; set; }
+    private string ChatName { get; set; }
+    private bool ChatLoaded { get; set; }
+
+    private User? _otherUser;
+
+    /// <summary>
+    ///     Null if the chat is a group chat
+    /// </summary>
+    private User? OtherUser
```

Comment:
noticing an issue here; this property doesn't adequately account for changes to the other user in the dm when switching DM views (i.e., the chat id parameter is set) the header remains the same, but the member list changes.